cookbook/B66_gadget.rst

```diff
@@ -0,0 +1,534 @@
+Recipe: GADGET Targets from Stern Batman '66
+============================================
+
+This guide shows you how to build an MPF config for *Batman 66's*
+GADGET targets. The idea is you can use this as a guide to 
+implement a similar feature in your machine.
+
+.. note::
+
+   This recipe requires MPF 0.53 or newer.
+
+This guide uses the following concepts in MPF:
+
+* :doc:`/game_logic/modes/index`
+* :doc:`/config_players/event_player.html`
+* :doc:`/game_logic/logic_blocks/accruals`
+* :doc:`events/overview/conditional.html`
+* :doc:`/config_players/show_player.html`
+* :doc:`/config_players/shows`
+
+
+**TODO**
+You can find the complete runnable machine config for this recipe in the
+``cookbook/B66_Gadget`` folder of the
+`mpf-examples repository <https://github.com/missionpinball/mpf-examples>`_
+on GitHub.
+
+What is GADGET mode?
+--------------------
+
+In Bataman '66, a player may hit each of the 6 stand-up targets representing the letters of the word "GADGET". When all letters have been hit, the player is awarded a "Gadget" which gives the players special ability in the game.
+
+Here are the specific rules we need to implement:
+
+**GADGET**
+
+* Each letter begin unlit
+* Letters become lit when hit individually
+* When an already-lit letter is hit, award if there is an unlit letter adjacent to the already-lit letter, award one adjacent letter. (Friendly Neighbor)
```

Comment:
`award` is used twice in this line. Maybe `and` instead?
```
* When an already-lit letter is hit, and if there is an unlit letter...
```

Reply:
Yep, too many edits and not enough re-proofing. How do I fix it?

Reply:
You can futher edit the files in your PR here: https://github.com/flamtime/mpf-docs/blob/patch-6/cookbook/B66_gadget.rst. As `patch-6` is the branch from this PR it will automatically be reflected here.